app/services/generate_membership_pdf.rb

```diff
@@ -1,19 +1,29 @@
-require 'render_anywhere'
-
 class GenerateMembershipPdf
-  include RenderAnywhere
+  TITLES = {
+      "inside_member":   'Tagfelvételi kérelem <br> rendes tagsághoz',
+      "outside_member":  'Tagfelvételi kérelem <br> külső tagsághoz',
+      "not_member":      'Kilépési nyilatkozat',
+      "inactive_member": 'Tagfelvételi kérelem <br> öreg tagsághoz'
+  }.freeze
```

Comment:
I think it's a bit cleaner like this.
```suggestion
  TITLES = {
      inside_member:   'Tagfelvételi kérelem <br> rendes tagsághoz',
      outside_member:  'Tagfelvételi kérelem <br> külső tagsághoz',
      not_member:      'Kilépési nyilatkozat',
      inactive_member: 'Tagfelvételi kérelem <br> öreg tagsághoz'
  }.freeze
```